getgather/mcp/brand/amazon.py

```diff
@@ -41,12 +41,13 @@ async def get_purchase_history(
     browser_profile = get_mcp_browser_profile()
     path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "patterns", "**/*.html")
     patterns = load_distillation_patterns(path)
-    purchases, _ = await run_distillation_loop(
+    _terminated, distilled, converted = await run_distillation_loop(
         f"https://www.amazon.com/your-orders/orders?timeFilter=year-{target_year}&startIndex={start_index}",
         patterns,
         browser_profile=browser_profile,
         stop_ok=True,
     )
+    purchases = converted if converted else distilled
     return {"purchases": purchases}
```

Comment:
While this is fine for now, I think eventually we want to be explicit to the caller.
Perhaps a future follow-up will need two keys for the return object, `purchases` on a successfully converted data, and something else (e.g. `raw_data` or a better name) if the conversation to JSON fails and thus this will contain the original HTML fragment.